targets/nordic_pca10095_net-blehci/syscfg.yml

```diff
@@ -39,4 +39,7 @@ syscfg.vals:
     BLE_MULTI_ADV_INSTANCES: 5
     BLE_PERIODIC_ADV: 1
     BLE_PERIODIC_ADV_SYNC_TRANSFER: 1
-    BLE_VERSION: 51
+    BLE_PERIODIC_ADV_SYNC_BIGINFO_REPORTS: 1
```

Comment:
@sjanc how to handle that?

Reply:
I think we should enable all (non-experimental) features in default netcore image, so this is fine but should be in separate PR

Reply:
No longer in current patch set.